crates/subspace-farmer/src/bin/subspace-farmer/main.rs

```diff
@@ -50,6 +50,13 @@ struct FarmingArgs {
     max_plot_size: Option<u64>,
 }
 
+#[derive(Debug, Clone, Copy, ArgEnum)]
+#[clap(default_value = "nothing")]
```

Comment:
Just curious, doesn't Clap support `Default`?

Reply:
Yeah, it does. Will change it :)

Reply:
Right now it doesn't work as intended